style.css

```diff
@@ -259,3 +279,56 @@ html, body {
     font-size: 1.5rem;
     font-family: "Fredericka the Great";
 }
+
+
+/* under here is modal */
```

Comment:
really? i wouldnt have guessed that from the class name of `modal`

Reply:
removed